source/dagger/core/input/inputs.cpp

```diff
@@ -14,8 +14,10 @@ using namespace dagger;
 
 void InputSystem::OnKeyboardEvent(KeyboardEvent input_)
 {
+    if ((SInt32)input_.key < 0) return;
```

Comment:
try to use {} even for one line, it's more readable that way 

{
    return;
}